capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -50,17 +50,16 @@ export class WikiMediaComponent implements OnInit, OnDestroy {
       let endIndex = firstPartOfString.indexOf("<h2>", 0);
```

Comment:
I seem to recall that some of the examples had image tags also.
Did you want to show those in your text too?

Reply:
No, we removed those since Billy will be showing his own images. I did add a section with img links from the bottom.